HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+## 1.0.3 / 2021-07-20
+
+- gitignore edit
```

Comment:
Maybe `Cleaned up gitignore`? I think that's what you said in most of these PRs.

Reply:
as discussed on the other PR, I went with 'edit' because it's a smaller change set rather than a complete rebuild of the file